aspnetcore/signalr/hubs.md

```diff
@@ -609,6 +612,76 @@ Using `Hub<IChatClient>` enables compile-time checking of the client methods. Th
 > [!NOTE]
 > The `Async` suffix isn't stripped from method names. Unless a client method is defined with `.on('MyMethodAsync')`, don't use `MyMethodAsync` as the name.
 
+## Client results
+
+In addition to making calls to clients, the server can request a result from the client. This requires the server to use `ISingleClientProxy.InvokeAsync` and the client to return a result from their `.On` handler.
```

Comment:
Going for number agreement on this one, plural "clients" throughout ...

```suggestion
In addition to making calls to clients, the server can request a result from clients. This requires the server to use `ISingleClientProxy.InvokeAsync` and the clients to return a result from their `.On` handlers.
```

Reply:
Using clients might convey that you can get results from multiple clients at a time with a single API, which isn't a supported scenario. I changed the text to say "the server can request a result from **a** client.". If you think it's better with "clients" still, or have a better alternative let me know.